eggo/dag.py

```diff
@@ -96,13 +132,13 @@ def _dnload_to_local_upload_to_s3(source, destination, compression):
 
         # 3. upload to tmp S3 location
         tmp_s3_path = os.path.join(EGGO_S3_TMP_URL, random_id())
-        upload_cmd = 'pushd {tmp_dir} && aws s3 cp ./* {s3_path} && popd'
+        upload_cmd = 'pushd {tmp_dir} && aws s3 cp . {s3_path} --recursive && popd'
```

Comment:
What's the rationale behind this change?  At the moment, I think that additional levels of nested directories will break our URL scheme.  But perhaps this is something we need to support?  cc @tomwhite